engines/made/screen.cpp

```diff
@@ -979,6 +979,10 @@ void Screen::clearSpriteList() {
 	_spriteList.clear();
 }
 
+void Screen::setMouseCursor(const Graphics::Cursor *cursor) {
+	CursorMan.replaceCursor(cursor, true);
```

Comment:
Could you add a comment explaining why `dontScale` is required here?

Reply:
a bit late now that it's been merged :P  the Windows cursors are much higher resolution than the MS-DOS ones and appear much larger than they should.  In winmade.exe the gameplay "canvas" is scaled up but the cursors are not.

Reply:
What resolution is the canvas normally scaled to?

Reply:
Display resolution (I think), and the cursors are not scaled to match.